linters/.jscsrc

```diff
@@ -23,10 +23,9 @@
   "disallowTrailingComma": true,
   "disallowTrailingWhitespace": true,
   "disallowYodaConditions": true,
-  "maximumLineLength": 80,
+  "maximumLineLength": 120,
   "requireBlocksOnNewline": 1,
   "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
```

Comment:
The reasoning is easy for people who think this is funny: Bad grammar makes me [sic]. I'm one of those people, but I'm allergic to it without the `exceptMultiline` value.